src/plugins/intel_cpu/src/emitters/snippets/x64/jit_binary_call_emitter.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2020-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "jit_binary_call_emitter.hpp"
+
+#include "emitters/plugin/x64/utils.hpp"
+
+using namespace Xbyak;
+using namespace dnnl::impl;
+using namespace dnnl::impl::cpu::x64;
+
+namespace ov {
+namespace intel_cpu {
+
+using jit_generator = dnnl::impl::cpu::x64::jit_generator;
+using cpu_isa_t = dnnl::impl::cpu::x64::cpu_isa_t;
+using ExpressionPtr = ov::snippets::lowered::ExpressionPtr;
+
+jit_binary_call_emitter::jit_binary_call_emitter(dnnl::impl::cpu::x64::jit_generator* h,
+                                                 dnnl::impl::cpu::x64::cpu_isa_t isa,
+                                                 std::set<snippets::Reg> live_regs)
+    : jit_emitter(h, isa),
+      m_regs_to_spill(std::move(live_regs)),
+      m_regs_initialized(false) {}
+
+void jit_binary_call_emitter::init_binary_call_regs(size_t num_binary_args,
+                                                    const std::vector<size_t>& in,
+                                                    const std::vector<size_t>& out) const {
+    std::vector<size_t> mem_ptr_idxs = in;
+    mem_ptr_idxs.insert(mem_ptr_idxs.end(), out.begin(), out.end());
+    init_binary_call_regs(num_binary_args, mem_ptr_idxs);
+}
+
+void jit_binary_call_emitter::init_binary_call_regs(size_t num_binary_args,
+                                                    const std::vector<size_t>& mem_ptrs_idxs) const {
```

Comment:
Minor: maybe to remote to `used_gprs` or `call_gprs` at least? I spent some time understand what is `mem_ptr_idxs`. The meaning is obvious when you us it in specific emitter: `jit_brgemm_emitter`, for example. But in bounds of `jit_binary_call_emitter` - it's not so obvious :)

Reply:
Sure, no problem. renamed to `used_gpr_idxs`